app/helpers/days_helper.rb

```diff
@@ -19,4 +19,13 @@ def day_status_css(d)
   def feedback_submitted?(day)
     student? && current_user.day_feedbacks.find_by(day: day.to_s)
   end
+
+  def total_cohort_students
+    @total_cohort_students ||= cohort.students.length
+  end
+
+  def completed_students(activity)
+    completed_students = User.students_completed_activity(activity, cohort)
```

Comment:
This line should use the `completed_activity` scope. Please replace this line with:

```ruby
completed_students = cohort.students.completed_activity(activity).count
```